src/divider/divider.tsx

```diff
@@ -16,16 +18,27 @@ export const Divider = ({
     lgCols,
     xlCols,
     xxlCols,
-    mobileCols,
-    tabletCols,
-    desktopCols,
+    mobileCols = 4,
+    tabletCols = 8,
+    desktopCols = 12,
     ...otherProps
 }: DividerProps) => {
+    const theme = useTheme();
     const isV2Layout =
         mobileCols !== undefined ||
         tabletCols !== undefined ||
         desktopCols !== undefined;
 
+    const colProps = {
+        xxsCols: xxsCols ?? Breakpoint["xxs-column"]({ theme }),
+        xsCols: xsCols ?? Breakpoint["xs-column"]({ theme }),
```

Comment:
now that the ColDiv can accept `-1`, we can use that as the default value instead of having to pull from the theme